tests/test_utils.py

```diff
@@ -29,10 +29,10 @@ def test_write_csv_ascii(generate_data, tmpdir):
     path = str(tmpdir.join('test_data.csv'))
     utils.write_csv(df, path)
 
-    with open(path, 'rb') as f:
+    with open(path, "r") as f:
```

Comment:
```suggestion
    with open(path, "rb") as f:
```

Reply:
explanation: in python 3 if you don't read as binary then "\r\n" becomes "\n", but if you read as binary then decode it works the same in 2 and 3.

Reply:
That makes sense, I took the opposite approach to convert everything `\r\n` to `\n` to fix the issue.